indico/modules/categories/models/categories.py

```diff
@@ -224,10 +225,9 @@ def get_root(cls):
     @property
     def url(self):
         if self.is_root:
-            return url_for('misc.index')
+            return url_for_index()
```

Comment:
`url_for('categories.display', self)` covers both cases; no need for the `if` anymore